shared/chat/inbox/row/big-team-header/container.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import {pausableConnect, type TypedState} from '../../../../util/container'
+import {BigTeamHeader} from '.'
+
+const mapStateToProps = (state: TypedState, {teamname, isActiveRoute}) => {
+  return {teamname, isActiveRoute}
+}
+
+const mapDispatchToProps = (dispatch, {navigateAppend}) => ({
```

Comment:
Not sure `navigateAppend` is making it through here. I get `TypeError: navigateAppend is not a function` when I try to manage my channels from the inbox.

Reply:
great catch! fixed